apps/web/src/components/templates/in-app-editor/FeedItems.tsx

```diff
@@ -0,0 +1,237 @@
+  field: any;
+}
+
+export function FeedItems(props: IFeedItemPopoverProps) {
+  const { data: feeds } = useQuery<FeedEntity[]>(QueryKeys.getFeeds, getFeeds);
+
+  return (
+    <FeedsBlock>
+      <Grid gutter={10}>
+        {(feeds || []).map((item, feedIndex) => {
+          return (
+            <Grid.Col span={4}>
+              <FeedPopover
+                field={props.field}
+                item={item}
+                feedIndex={feedIndex}
+                showFeed={props.showFeed}
+                index={props.index}
+                setValue={props.setValue}
+              />
+            </Grid.Col>
+          );
+        })}
+      </Grid>
+    </FeedsBlock>
+  );
+}
+
+function FeedPopover(props: IFeedPopoverProps) {
+  const [opened, setOpened] = useState(false);
+  const colorScheme = useMantineTheme().colorScheme;
+
+  return (
+    <Popover
+      opened={opened}
+      onClose={() => setOpened(false)}
+      target={
+        <FeedChip
+          item={props.item}
+          feedIndex={props.feedIndex}
+          setOpened={setOpened}
+          index={props.index}
+          showFeed={props.showFeed}
+          field={props.field}
+          setValue={props.setValue}
+          onEditClick={() => {
+            setOpened((prevCheck) => !prevCheck);
+          }}
+        />
+      }
+      width={260}
+      position={'bottom'}
+      placement={'center'}
+      withArrow
+      styles={{
+        root: {
+          width: '100%',
+          height: '1px',
+        },
+        inner: { margin: 0, padding: 0, height: '95px' },
+        arrow: {
+          backgroundColor: colorScheme === 'dark' ? colors.B20 : colors.white,
+          height: '-22px',
+          border: 'none',
+          margin: '0px',
+          top: '-3px',
+        },
+        body: {
+          backgroundColor: colorScheme === 'dark' ? colors.B20 : colors.white,
+          color: colorScheme === 'dark' ? colors.white : colors.B40,
+          border: 'none',
+          marginTop: '1px',
+          width: '100%',
+        },
+        popover: { width: '170px', height: '95px' },
+      }}
+    >
+      <PopoverActionBlock setOpened={setOpened} showFeed={props.showFeed} feedItem={props.item} />
+    </Popover>
+  );
+}
+
+function PopoverActionBlock({
+  setOpened,
+  showFeed,
+  feedItem,
+}: {
+  setOpened: (boolean) => void;
+  showFeed: boolean;
+  feedItem?: FeedEntity;
+}) {
+  const colorScheme = useMantineTheme().colorScheme;
+
+  return (
+    <ActionBlockWrapper colorScheme={colorScheme} onMouseLeave={() => setOpened(false)}>
+      <CopyBlock showFeed={showFeed} feedItem={feedItem} />
+      <DeleteBlock setOpened={setOpened} showFeed={showFeed} feedItem={feedItem} />
+    </ActionBlockWrapper>
+  );
+}
+
+function CopyBlock({ showFeed, feedItem }: { showFeed: boolean; feedItem?: FeedEntity }) {
+  const [opened, setOpened] = useState(false);
+
+  const colorScheme = useMantineTheme().colorScheme;
```

Comment:
```suggestion
  const { colorScheme } = useMantineTheme();
```

I think it's nicer like this wdyt?